client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -118,6 +120,26 @@ class EditContactInfoFormCard extends React.Component {
 		}
 	}
 
+	hasEmailChanged() {
+		return this.props.contactInformation.email === formState.getFieldValue( this.state.form, 'email' );
+	}
+
+	getCurrentEmails() {
+		const currentEmail = this.props.contactInformation.email,
+			wpcomEmail = this.props.currentUser.email;
+
+		if ( currentEmail === wpcomEmail ) {
+			return currentEmail;
+		}
+
+		return this.props.translate( '%(currentEmail)s (additionally to %(wpcomEmail)s)',
+			{
+				args: { currentEmail, wpcomEmail },
+				context: 'List of emails the WHOIS confirmation email is sent to'
```

Comment:
@breezyskies, it's in both screenshots above :) It's just part of the larger sentence. It's constructed separately since we mention one email (if they are the same) and two if the wpcom one differs (so the wpcom one becomes the backup email). See `p7DVsv-1TW-p2` for more details :)